.pre-commit-config.yaml

```diff
@@ -24,6 +24,7 @@ repos:
     rev: 7.1.0
     hooks:
       - id: flake8
```

Comment:
Here's another hook for pyproject.toml files

https://github.com/marksweb/django-nh3/blob/main/.pre-commit-config.yaml#L24

I've probably got rstcheck in there as well so that'd be worth adding

Reply:
Cool, I've added both!